packages/shared/src/graphql/common.ts

```diff
@@ -119,7 +119,7 @@ export const DEFAULT_ERROR = 'An error occurred, please try again';
 
 export const errorMessage = {
   profile: {
-    invalidUsername: 'Invalid characters found in username!',
+    invalidUsername: 'Username can only contain Alphanumeric (A-z, a-z, 1234..)',
```

Comment:
There is actually a specific regex condition we use for checking usernames, and it is not limited to this.

Reply:
Ok, so it is automated or can be changed?

Reply:
![image](https://github.com/user-attachments/assets/f4fe2866-3728-4704-967f-51655fa8e192)
This one but it is checking the condition and the issue was just showcasing the error message isn't these both are different? Please clear my doubt.

Reply:
The one we use for usernames is the `handleRegex`

Reply:
Thanks Updating and solving issue!

Reply:
The message is still inaccurate like mentioned.

Reply:
Sorry, due to some other work I was busy solving it in 1 hour.

Reply:
When i am going to testuser profile page without doing any changes I am facing this below mentioned issue
![image](https://github.com/user-attachments/assets/9a6d4037-901d-45fc-989b-5b498c0d4849)
and in code I am facing this issue:
![Screenshot 2024-10-15 122504](https://github.com/user-attachments/assets/8d3b3751-8f76-49e4-880d-8db3656b83dc)

Reply:
All cool, no worries. Thank you for trying! 🙏